### PR TITLE
Feature/js problems =>main (pull request)

### PR DESCRIPTION
**Pull request by AliHassanZulfiqar**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
```
Comment (sheerryy) [inline .idea/.gitignore:2]: Add `.idea` folder to gitignore

File: hackerRankProblemSolving/veryBigArraySum/veryBigArraySum.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function simpleArraySum(ar) {
+  let sum = 0;
+
+   for(let i = 0; i < ar.length; i++) {
+       sum = ar.reduce((a, b) => a + b);
```
Comment (sheerryy) [inline hackerRankProblemSolving/veryBigArraySum/veryBigArraySum.js:5]: NICE!

**Review by sheerryy — COMMENTED** [review]
Awesome

File: .idea/.gitignore (hunk 1 above)
Comment (AliHassanZulfiqar) [reply .idea/.gitignore:2]: Okay, will do it.

File: hackerRankProblemSolving/veryBigArraySum/veryBigArraySum.js (hunk 2 above)
Comment (AliHassanZulfiqar) [reply hackerRankProblemSolving/veryBigArraySum/veryBigArraySum.js:5]: Thank you.

**Pull request merged**